City-of-Helsinki/helsinki-paatokset
# PP-165: PP-165: created a block for all initiatives and reused styles…

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-165

Cerated a block for all initiatives that will render to a page with path /kaikki-aloitteet

**How to test**

Checkout branch and run `drush cim -y; drush cr`. Create a page that has /kaikki-aloitteet as its path. The block should render there with all the initiatives that exists. Styles are the same as in policymaker documents listing.

Changed the document listing so that the whole row is a link and added hover effect to it.
